### DISCUSSION TITLE
async_parallelize how to use

### DISCUSSION THREAD

**Discussion by davidxiaozhi** [modular/modular#1745] [Q&A]
in mojo ,how to use the async_parallelize method? is there any relevant case or demo use ?

**Comment by stefanuytterhoeven** [comment]
Would like to know that too. I've read mojo supports it, but would like to see examples. Simple examples

**Comment by hcye2022** [comment]
The API reference shows only sync_parallelize and parallelize but not async_parallelize. parallelize seems to be async, with this example: https://github.com/rd4com/mojo-learning/blob/main/tutorials/multi-core-parallelize-with-simd%20.md. Not sure what sync_parallelize does?
